dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -33,6 +34,7 @@
  *
  * @author dyuen
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Hi,
Please investigate what changed. Is workingDirectory a recent addition? Maybe the swagger library is changing without us noticing (is the version pinned?) As it stands, this seems really odd.